docs/pages/installation/docker-extension.md

```diff
@@ -100,7 +103,7 @@ If you want to remove Meshery as a Docker extension from your system, follow the
    - Meshery might have pulled Docker images for its components. You can remove these images using the `docker rmi` command. Replace the image names with the actual ones you want to remove:
 
    <pre class="codeblock-pre" style="padding: 0; margin-top: 2px; font-size:0px;"><div class="codeblock" style="display: block;">
-    <div class="clipboardjs" style="padding: 0">
+    <div class="clipboardjs" style="padding: 0; height: 8px; overflow: hidden;">
```

Comment:
I know, the usage of `px` has been in practice for a long on these pages, but can we get rid of them as we add new styles?

Reply:
@sandramsc the implied suggestion here is to use `rem` as the unit of measure.

P.S. 

`0.5rem` is equivalent to `8px`